code/modules/mob/living/carbon/xenomorph/castes/hunter/abilities_hunter.dm

```diff
@@ -685,3 +747,169 @@
 	owner.playsound_local(owner, 'sound/effects/alien/new_larva.ogg', 25, 0, 1)
 	cooldown_duration = initial(cooldown_duration) //Reset the cooldown timer to its initial state in the event of a whiffed Silence.
 	return ..()
+
+//assassin things
+
+// ***************************************
+// *********** Lunge
+// ***************************************
+
+/datum/action/ability/activable/xeno/pounce/lunge
+	name = "Lunge"
+	action_icon = 'icons/Xeno/actions/hunter.dmi'
+	action_icon_state = "assassin_lunge"
+	desc = "Swiftly lunge at your destination, if on a target, attack them."
+	ability_cost = 10
+	cooldown_duration = 6 SECONDS
+	keybinding_signals = list(
+		KEYBINDING_NORMAL = COMSIG_XENOABILITY_HUNTER_LUNGE,
+	)
+	use_state_flags = ABILITY_USE_BUCKLED
+
+//Triggers the effect of a successful pounce on the target.
+/datum/action/ability/activable/xeno/pounce/lunge/trigger_pounce_effect(mob/living/living_target)
+	playsound(get_turf(living_target), 'sound/voice/alien/roar4.ogg', 25, TRUE)
+	var/mob/living/carbon/xenomorph/xeno_owner = owner
+	xeno_owner.UnarmedAttack(living_target)
+	step_away(living_target, xeno_owner, 1, 3)
+	xeno_owner.face_atom(living_target)
+
+// ***************************************
+// *********** Phase Out
+// ***************************************
+
+/datum/action/ability/xeno_action/stealth/phaseout
+	name = "Phase Out"
+	action_icon_state = "hunter_invisibility"
+	action_icon = 'icons/Xeno/actions/hunter.dmi'
+	desc = "Become fully invisible for 6 seconds, or until damaged. Attacking does not break invisibility."
+	ability_cost = 10
+	keybinding_signals = list(
+		KEYBINDING_NORMAL = COMSIG_XENOABILITY_TOGGLE_PHASEOUT,
+	)
+	cooldown_duration = 6 SECONDS
+	stealth_duration = 6 SECONDS
+	disable_on_signals = list(
+		COMSIG_LIVING_IGNITED,
+		COMSIG_LIVING_ADD_VENTCRAWL,
+	)
+	stealth_flags = DIS_POUNCE_SLASH
+	sneak_attack_armor_pen = 30
+
+// i just realized, we need to disable processing, this will inherently just amke handle_stealth not do anything
+///Updates or cancels stealth
+/datum/action/ability/xeno_action/stealth/phaseout/handle_stealth()
+	var/mob/living/carbon/xenomorph/xenoowner = owner
+	xenoowner.alpha = HUNTER_STEALTH_STILL_ALPHA * stealth_alpha_multiplier // instant full stealth regardless of movement.
+
+// ***************************************
+// *********** Death Mark
+// ***************************************
+/datum/action/ability/activable/xeno/hunter_mark/assassin
+	name = "Death Mark"
+	action_icon_state = "death_mark"
+	action_icon = 'icons/Xeno/actions/hunter.dmi'
+	desc = "Psionically disturb a creature for 15 seconds, allowing you to deal a stronger sneak attack. They will know you are coming for them."
+	ability_cost = 50
+	keybinding_signals = list(
+		KEYBINDING_NORMAL = COMSIG_XENOABILITY_HUNTER_MARK,
+	)
+	cooldown_duration = 30 SECONDS
+	require_los = FALSE
+
+	///Duration for the mark.
+	var/timeout = 15 SECONDS
```

Comment:
probably just make this a define, there's no real reason for it to be a var since it never changes.